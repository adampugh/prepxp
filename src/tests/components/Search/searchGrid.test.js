import React from "react";
import { shallow } from "enzyme";

import SearchGrid from "../../../components/Search/searchGrid";
import searchLists from "../../fixtures/searchLists";

describe("<SearchGrid />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<SearchGrid filteredLists={searchLists.filteredLists}/>);
        expect(wrapper).toMatchSnapshot();
    });
});