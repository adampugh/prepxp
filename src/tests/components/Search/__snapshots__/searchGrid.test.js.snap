// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchGrid /> should match snapshot 1`] = `
<div
  className="searchGrid__lists"
>
  <div
    className="container-s"
  >
    <div
      className="searchGrid__list__item"
      key="Angular Developer0"
    >
      <div>
        <h2>
          Angular Developer
        </h2>
        <p>
          4
           Questions
        </p>
      </div>
      <h2
        className="searchGrid__list__icon"
      >
        <i
          className="fas fa-plus"
        />
      </h2>
    </div>
    <div
      className="searchGrid__list__item"
      key="Node JS Developer1"
    >
      <div>
        <h2>
          Node JS Developer
        </h2>
        <p>
          3
           Questions
        </p>
      </div>
      <h2
        className="searchGrid__list__icon"
      >
        <i
          className="fas fa-plus"
        />
      </h2>
    </div>
    <div
      className="searchGrid__list__item"
      key="General Interview Questions 12"
    >
      <div>
        <h2>
          General Interview Questions 1
        </h2>
        <p>
          7
           Questions
        </p>
      </div>
      <h2
        className="searchGrid__list__icon"
      >
        <i
          className="fas fa-plus"
        />
      </h2>
    </div>
  </div>
</div>
`;
